Lib/test/test_eof.py

```diff
@@ -37,7 +37,8 @@ def test_EOFS_with_file(self):
         self.assertIn(b'unterminated triple-quoted string literal (detected at line 3)', err)
 
     def test_eof_with_line_continuation(self):
-        expect = "unexpected EOF while parsing (<string>, line 1)"
+        expect = "(unicode error) 'unicodeescape' codec can't decode bytes in " \
```

Comment:
This looks odd, I would prefer to change the invalid Unicode escape to something that still produces the EOF for example:

```
compile('"\\Xhh" \\', "string", "exec")
```

Reply:
Thanks! I have applied your suggestion and the test case has passed. 
But there is a new warning, I guess this is not a expected behavior.
```
test_eof_with_line_continuation (Lib.test.test_eof.EOFTestCase.test_eof_with_line_continuation) ... <string>:1: SyntaxWarning: invalid escape sequence '\X'
ok
```

Reply:
You can add a suppression context manager (we should do it anyway because otherwise the test suite will fail with `-Werror`